pytensor/link/numba/dispatch/extra_ops.py

```diff
@@ -186,7 +186,7 @@ def ravelmultiindex(*inp):
 
             new_arr = arr.T.astype(np.float64).copy()
             for i, b in enumerate(new_arr):
-                for j, (d, v) in enumerate(zip(shape, b)):
+                for j, (d, v) in enumerate(zip(shape, b, strict=True)):
```

Comment:
possibility of messing with JIT

Reply:
This one indeed messes with the JIT, numba doesn't want any `strict` argument (either `True` or `False`). The others seem to pass? Or are not tested?

Reply:
Maybe not tested